cms/templates/export_git.html

```diff
@@ -49,9 +49,9 @@ <h2 class="title">${_("Export Course to Git:")}</h2>
       <div class="messages">
         % if msg:
           % if failed:
-          <h3 class="error-text">${_('Export Failed')}:</h3>
+          <h3 class="error-text">${_('Export Failed:')}</h3>
```

Comment:
We're fairly inconsistent with the colons around our codebase. Why is this change needed? I don't love the idea of making the translators re-translate this just to add a colon.